<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;奇怪的加密软件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;对称加密算法设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Hash设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Hash_horizontalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;算法：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="SHAradioButton">
      <property name="text">
       <string>SHA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="MD5radioButton">
      <property name="text">
       <string>MD5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;加密目标&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始加密</string>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>307</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="key_horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;密钥：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="random_radioButton">
      <property name="text">
       <string>随机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="self_radioButton">
      <property name="text">
       <string>自定义</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="key_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SEA_horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;算法：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="DESradioButton">
      <property name="text">
       <string>DES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="AESradioButton">
      <property name="text">
       <string>AES</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="str_horizontalLayout">
    <item>
     <widget class="QRadioButton" name="str_radioButton">
      <property name="text">
       <string>字符串</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="str_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="file_horizontalLayout">
    <item>
     <widget class="QRadioButton" name="file_radioButton">
      <property name="text">
       <string>文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="file_pushButton">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
